5.5/root/usr/share/container-scripts/mysql/common.sh

```diff
@@ -5,8 +5,14 @@
 # volume to override it.
 export MYSQL_DATADIR=/var/lib/mysql/data
 
+# Get prefix rather than hard-code it
+export MYSQL_PREFIX=$(which mysqld_safe|sed -e 's|/bin/mysqld_safe$||')
```

Comment:
This way we can share the scripts across the versions and SCL vs. non-SCL images. `which` RPM is only 77kB, does it seem to be worth the trouble of maintaining different versions of scripts to save that little space? From my PoV it does not.. But maybe we can avoid using which somehow.. just not having a quick idea right now.